### PR TITLE
Rebase to latest upstream/master

### PR DESCRIPTION
**Pull request by nosovmik**

### CODE REVIEW

File: inference-engine/src/inference_engine/ie_core.cpp [hunk 1]
```diff
@@ -205,6 +206,7 @@ class Core::Impl : public ICore {
 
     std::unordered_set<std::string> opsetNames;
     std::vector<IExtensionPtr> extensions;
+    ngraph::frontend::FrontEndManager frontEndManager;
```
Comment (slyalin) [inline inference-engine/src/inference_engine/ie_core.cpp:209]: why?

Comment (nosovmik) [reply inference-engine/src/inference_engine/ie_core.cpp:209]: This is for future implementation of 'ReadNetwork'

**Review by slyalin — CHANGES_REQUESTED** [review]
Wanted to merge and just realized that the target branch is slyalin:master, but should be pdpd_frontend.

**Comment by nosovmik** [comment]
Changed target branch, sorry

**Pull request merged**